indico/modules/events/registration/controllers/display_test.py

```diff
@@ -43,3 +46,50 @@ def test_RHRegistrationForm_can_register(db, dummy_regform, dummy_reg, dummy_use
     assert rh._can_register()  # withdrawn/rejected do not count against limit
     dummy_reg.state = RegistrationState.complete
     assert not rh._can_register()  # exceeding limit
+
+
+@pytest.mark.usefixtures('request_context')
+def test_RHRegistrationWithdraw(smtp, dummy_regform, dummy_reg, dummy_user):
```

Comment:
not a big fan of the uppercase stuff in the name :) `test_withdraw_registration_rh`?

PS: thanks for adding a test!